src/Core/Services/AuthService.cs

```diff
@@ -476,6 +485,11 @@ public async Task<PasswordlessLoginResponse> GetPasswordlessLoginRequestByIdAsyn
             return await _apiService.GetAuthRequestAsync(id);
         }
 
+        public async Task<PasswordlessLoginResponse> GetPasswordlessLoginResponse(string id, string accessCode)
+        {
+            return await _apiService.GetAuthResponseAsync(id, accessCode);
+        }
```

Comment:
Is this really needed, can't the caller directly use the `IApiService` ?

Reply:
I left it to be easier for a dev who is using Passwordless methods to have them all in one place and with intuitive naming.